CriticalMass/AlertPresenter.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Created for CriticalMaps in 2020
+
+import UIKit
+
+final class AlertPresenter {
+    private var viewController: UIViewController? {
+        UIApplication.shared.keyWindow?.rootViewController
+    }
+
+    static let shared = AlertPresenter()
+
+    private init() {}
+
+    func presentAlert(
+        title: String?,
+        message: String? = nil,
+        preferredStyle: UIAlertController.Style,
+        actionData: [UIAlertAction],
+        isCancelable: Bool = false
+    ) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: preferredStyle)
+        actionData.forEach(alert.addAction(_:))
+        if isCancelable {
+            let cancelAction = UIAlertAction(title: "Cancel", style: .cancel)
+            alert.addAction(cancelAction)
+        }
+        onMain {
+            self.viewController?.present(alert, animated: true, completion: nil)
+            UIAccessibility.post(notification: .announcement, argument: nil)
```

Comment:
I don't think that is needed. UIAlertController does that automatically.

Reply:
Wasn’t sure if it does. Is there and place those things get documented?

Reply:
I'm not aware of any documentation about the Accessibility Notifications. 

A good approach is to test the App first with assistive technologies and then add notifications if needed. Also the .announcement notification should only be used if you want that the screenreader announces text. If you want to notify the assistive tech that the visible UI elements changed you can use .layoutChanged or .screenChanged.